pcs_test/tier1/legacy/test_resource.py

```diff
@@ -2980,6 +2980,13 @@ def testGroupPromotableCreation(self):
 
     @skip_unless_crm_rule()
     def testGroupRemoveWithConstraints1(self):
+        # The mock binary for crm_resource cannot perform the `move-with-constraint` command.
+        # This was not a problem before cleanup of paths to binaries, since the move command
+        # did not use the path from settings.crm_resource_exec, but path that was constructed in the command.
+        # After cleanup, every command uses paths from the settings,
+        # so we can't use the path to the mock binary for this test anymore.
```

Comment:
We don't really need to know why this was working before. And the comment misses context needed to understand it. Let's just write that the mock executable doesn't support the `move-with-constraint` command and therefore the real executable is used. Also, please wrap the comment to 80 chars per line.